src/pages/community/faq.md

```diff
@@ -23,6 +23,12 @@ They'll take your code from the (macOS) clipboard, convert it, and paste it back
 #### What's the relation between Reason, BuckleScript and OCaml?
 See [here](/guide/javascript). Reason's a syntax for OCaml and supports all its features. BuckleScript compiles OCaml/Reason code into JavaScript.
 
+#### How does Reason (and BuckleScript) compare to Elm
+Reason is a general-purpose language that can target node, native code and utilize both the opam and npm ecossytems. Elm is more opininated and focused, aiming at improving the experience in its own niche and ecosystem, at a rather high cost to its flexibility. Elm has better error messages, though Reason is closing in fast. BuckleScript has superb js interop, generates highly readable and easily debuggable js code and can utilize the npm ecosystem to its full extent, while Elm's FFI and interop is rather convoluted. Elm is pure, while Reason is pragmatic (take that as you wish :)). OCaml has had 25 or so years to mature, and has an active academic base of contributors that keep it close to the forefront of progamming langauge development, while Elm is relatively new and unstable, but moves faster. Reason has first-class support for React, while Elm is focused solely on "The Elm Architecture" (TEA). Reason does however also have a TEA implementation that does a surprisingly good job of mimicing its muse.
```

Comment:
> Reason does however also have a TEA implementation that does a surprisingly good job of mimicing its muse.

Yay!  ^.^

Reply:
Two suggestions on the Elm section:

* Might as well link to https://github.com/OvermindDL1/bucklescript-tea :)
* "interop is rather convoluted" might be interpreted as combatative– even though I would personally agree with the statement, it might be more diplomatic to use a relative phrase like "is more involved" or "is more complex"

Reply:
@jaredly 
* I've tried to integrate your first point with some other points I was making, while being a bit briefer.
* Immutable vs mutable data types is covered by the "pure vs non-pure/pragmatic" point I think, which I realize you'd rather remove, but this is a bit wordy and specific, without addressing the other aspects of purity (which will quickly get you into controversial territory).
* "pragmatic" is admittedly a bit of a weasel word, but Andre completely evacuates the meaning of the words (maybe that was the point though). My intention here is to address what is a very important distinction, acknowledge that the topic is controversial, and at the same time keep it brief. That has to come at some expense to precision. I think being vague is ok as long as we're honest and open about it. It can probably be improved, but I don't think it should just be removed.